packages/manager/apps/ips/src/pages/listing/ipListing/components/DatagridCells/IpActionsCell/IpActionsCell.tsx

```diff
@@ -20,6 +20,34 @@ export type IpActionsCellParams = {
   ip: string;
   parentIpGroup?: string;
 };
+/* 
+  list of category and path map in order to check if the ip is attached to that category
+  categoryAndPathMapping = [
+        {
+            category: "CLOUD",
+            path: "/cloud/project",
+        },{
+            category: "VRACK",
+            path: "/vrack",
+        },{
+            category: "DEDICATED",
+            path: "/dedicated/server"
+        },{
+            category: "HOUSING",
+            path: "/dedicated/housing",
+        },{
+            category: "PRIVATE_CLOUD",
+            path: "/dedicatedCloud"
+        },{
+            category: "VPS",
+            path: "/vps",
+        },{
+            category: "IP_LOAD_BALANCING",
+            path: "/ipLoadbalancing",
+        }
+    ];
+
+*/
```

Comment:
Not used finally ?

Reply:
Its for reference comment. We need this mapping.